.github/workflows/merge-to-stage.js

```diff
@@ -0,0 +1,233 @@
+const {
+  slackNotification,
+  getLocalConfigs,
+  pulls: { addLabels, addFiles, getChecks, getReviews },
+} = require('./helpers.js');
+
+// Run from the root of the project for local testing: node --env-file=.env .github/workflows/merge-to-stage.js
+const prTitle = '[Release] Stage to Main';
+const seen = {};
+const requiredApprovals = process.env.REQUIRED_APPROVALS || 2;
+const stage = 'stage';
+const prod = 'main';
+const labels = {
+  highPriority: 'high priority',
+  readyForStage: 'Ready for Stage',
+  SOTPrefix: 'SOT',
+};
+
+const slack = {
+  fileOverlap: ({ html_url, number, title }) =>
+    `:fast_forward: Skipping <${html_url}|${number}: ${title}> due to overlap in files.`,
+  merge: ({ html_url, number, title }) =>
+    `:merged: Stage merge PR <${html_url}|${number}: ${title}>.`,
+  failingChecks: ({ html_url, number, title }) =>
+    `:x: Skipping <${html_url}|${number}: ${title}> due to failing checks`,
+  requireApprovals: ({ html_url, number, title }) =>
+    `:x: Skipping <${html_url}|${number}: ${title}> due to insufficient approvals`,
+  openedSyncPr: ({ html_url, number }) =>
+    `:fast_forward: Created <${html_url}|Stage to Main PR ${number}>`,
+};
+
+let github, owner, repo, currPrNumber, core;
+
+let body = `
+## common base root URLs
+**Homepage :** https://www.stage.adobe.com/
+**BACOM:** https://business.stage.adobe.com/fr/
+**CC:** https://www.stage.adobe.com/creativecloud.html
+**Blog:** https://blog.stage.adobe.com/
+**Acrobat:** https://www.stage.adobe.com/acrobat/online/sign-pdf.html
+
+**Milo:**
+- Before: https://main--milo--adobecom.hlx.live/?martech=off
+- After: https://stage--milo--adobecom.hlx.live/?martech=off
+`;
+
+const RCPDates = [
```

Comment:
Hmm, maybe just a calendar entry for milo core?

Reply:
@mokimo  @overmyheadandbody  we could add a check if the current year is not 2024 anymore -> stop automerging completely.
this will ensure we will not forget to update a year :)

Reply:
Done